packages/neuron-wallet/src/utils/deep-camelize-keys.ts

```diff
@@ -0,0 +1,27 @@
+function snakeToCamel(key: string) {
```

Comment:
It should be specific for neuron data, including converting the value of a field, to avoid patching other methods, such as https://github.com/nervosnetwork/neuron/pull/3169/files#diff-f8aa9b4d7866ef60047f1dd4182bbb561a986750fbeb42785020bb98b738641fR18

```
 const _depType = depType == 'dep_group' ? DepType.DepGroup : depType
```

The converter should be cohesive, e.g. https://explorer.nervos.org/tools/snake-case-and-camel-case